content/en/docs/project-user-guide/application-workloads/services.md

```diff
@@ -1,10 +1,167 @@
+
+- **Stateful Service**
+
+  Create a Stateless Service also create a **Statefulset** workload.
+
+  For more information **Statefulset**, please visit [Statefulset](../../application-workloads/statefulsets)
+
+- **External Service**
+
+  The service created by External Service is a service internet access method. It is different from Stateless and stateful.
+
+- **Specify Workloads**
+
+  Create a service with existing Pods.
+
+- **Edit by YAML**
+
+  Create directly with Yaml, you can upload and download Yaml configuration files
+
+{{< notice tip>}}
+
+  The value of `annotations:kubesphere.io/serviceType` keywords is one of them **statelessservice** , **statefulservice** , **externalservice** and **None**.
+
+{{</ notice>}}
+
+## Create a **Stateless Service**
+
+Into **Application Workload**, click **Services**
+
+### Step 1: click the **Create**, choose **Stateless Service**
+
+![services_lists](/images/docs/project-user-guide/workloads/services_lists.png)
+
+![stateless_form](/images/docs/project-user-guide/workloads/stateless_form.png)
+
+### Step 2: Input Base data
+
+![stateless_form_1](/images/docs/project-user-guide/workloads/stateless_form_1.png)
+
+- **Name**: The name of the Service and Deployment, which is also the unique identifier.
+- **Alias**: The alias name of the service, making resources easier to identify.
+- **Version**: It can only contain lowercase letters and numbers. The maximum length of characters is set to 16.
+
+{{< notice tip >}}
+
+**Name** have named two configs, `Deployment` and `Service`.
+
+{{</ notice>}}
+
+``` yaml
+kind: Deployment
+metadata:
+  labels:
+    version: v1
+    app: xxx
+  name: xxx-v1
+spec:
+  selector:
+    matchLabels:
+      version: v1
+      app: xxx
+  template:
+    metadata:
+      labels:
+        version: v1
+        app: xxx
+---
+kind: Service
+metadata:
+  labels:
+    version: v1
+    app: xxx
+  name: xxx
+spec:
+    metadata:
+      labels:
+        version: v1
+        app: xxx
+```
+
+### Step 3: Input Container Image
+
+![stateless_form_2.png](/images/docs/project-user-guide/workloads/stateless_form_2.png)
+
+To add a container for the service, please refer to [Pod Containers](../deployments) for details.
+
+### Step 4: Input Volume
+
+![stateless_form_3](/images/docs/project-user-guide/workloads/stateless_form_3.png)
+
+Refer to [Pod Volumes](../deployments) for details.
+
+### Step 5: input Advanced Settings
+
+![stateless_form_4](/images/docs/project-user-guide/workloads/stateless_form_4.png)
+
+Refer to [Deployment Advanced Settings](../deployments) for details.
+
+## Check Service Detail
+
+### Service Operations
+
+![stateless_finish](/images/docs/project-user-guide/workloads/services_lists_finish.png)
+
+Now the Stateless Service has created, you can click it to view more information about it.also you can click **More** to operate this service item include edit config and delete.
```

Comment:
->
After the stateless service is created, you can click the three dots on the right to further edit it, such as its metadata (excluding **Name**), YAML, port, and Internet access.

Reply:
done